default.xml

```diff
@@ -4,16 +4,20 @@
     <remote name="prpl" fetch="https://github.com/prplfoundation/" pushurl="ssh://git@github.com/prplfoundation" />
     <remote name="nanomsg" fetch="https://github.com/nanomsg/" />
     <remote name="rurban" fetch="https://github.com/rurban/" />
+    <remote name="boardfarm" fetch="https://github.com/mattsm/" />
```

Comment:
Why are we using a fork?

Reply:
It's not a fork, as boardfarm's main repo(_qca/boardfarm_) moved to _mattsm/boardfarm_.